pages/dashboard.vue

```diff
@@ -80,12 +96,29 @@ export default {
         },
       ]
     } else {
-      products = await getAccountProducts()
+      products = await fetchAccountProducts(await this.getAccountId())
     }
     this.subscribedProducts = products.filter(
-      product => product.subscriptionStatus === ProductStatus.ACTIVE)
-  }
-}
+      product => product.subscriptionStatus === ProductStatus.ACTIVE
+    )
+  },
+  methods: {
+    /**
+     * Returns account id from object in session storage.
+     * Waits up to 1 sec for current account to be synced (typically by SbcHeader).
+     */
+    async getAccountId (): Promise<number> {
```

Comment:
is this something that we should be doing in ppr too? Or is it just because it's slow in this project sometimes?